src/test/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GenerateVCFFromPosteriorsIntegrationTest.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+/**
+ * Integration test for {@link GenerateVCFFromPosteriors}
+ */
+public class GenerateVCFFromPosteriorsIntegrationTest extends CommandLineProgramTest {
+
+    // Test directory
+    private static final File TEST_DIR = new File(toolsTestDir, "copynumber/gcnv/");
+
+    private static final String SAMPLE_NAME = "SAMPLE_0";
+
+    private static final File VCF_CORRECT_OUTPUT = new File(TEST_DIR, "output.vcf");
+
+    @Test(expectedExceptions = UserException.BadInput.class)
+    public void testWrongInputChunkOrder() {
+        final List<String> chunkDirectoriesNames = new ArrayList<>(Arrays.asList(
+                "/normal_cohort_chunk_2-calls/", "/normal_cohort_chunk_0-calls/", "/normal_cohort_chunk_1-calls/"));
+        final List<String> chunkDirectoriesFileList = chunkDirectoriesNames.stream()
+                .map(s -> TEST_DIR.getAbsolutePath().concat(s)).collect(Collectors.toList());
+        final File outputVCF = createTempFile("test", ".vcf");
+
+        final List<String> arguments = new ArrayList<>();
+        chunkDirectoriesFileList.stream().forEach(dir -> {
+            arguments.add("-" + GenerateVCFFromPosteriors.POSTERIOR_CALL_DIRECTORY_FULL_NAME); arguments.add(dir);
+        });
+        arguments.add("-" + GenerateVCFFromPosteriors.SAMPLE_DIRECTORY_NAME_FULL_NAME);
```

Comment:
I think using an ArgsBuilder is cleaner.

Reply:
I'm not sure what ArgsBuilder is..